frontend/src/app/task-area/task-area.component.html

```diff
@@ -1,14 +1,22 @@
 <div class="task-area">
-  <h1>All Tasks</h1>
-  <input class="new-input" type="text" placeholder="Add a new task insdie ‘All’ category" />
+  <h1>{{ currentTab }} Tasks</h1>
+
+  <input (keyup.enter)="createTask()" [formControl]="taskText" class="new-input" type="text"
+    placeholder="Add a new task insdie '{{ currentTab }}' category" [ngClass]="{
+      invalid: taskText.errors && (taskText.dirty || taskText.touched),
+      valid: !taskText.errors && (taskText.dirty || taskText.touched)
+    }" />
+  <label class="error-message" *ngIf="taskText.invalid && taskText.value">This field invalid</label>
 
   <ng-template #taskTemplate let-itemData="item">
-    <input type="checkbox" [checked]="itemData.status" class="custom-checkbox" id="{{ itemData.id }}" />
-    <label for="{{ itemData.id }}"></label>
+    <label class="custom-checkbox" (click)="checkTask(itemData)">
```

Comment:
For what using `label` and `span class="checkmark"`?

Reply:
for painting checkmark sign

Reply:
Can you customize input with `::before` and `::after`? Without any extra tags in html?